tests/p2p/test_connections.py

```diff
@@ -33,8 +33,8 @@ def test_manager_connections(self):
         endpoint = 'tcp://127.0.0.1:8005'
         manager.connections.connect_to(endpoint, use_ssl=True)
 
-        self.assertNotIn(endpoint, manager.connections.connecting_peers)
-        self.assertNotIn(endpoint, manager.connections.handshaking_peers)
-        self.assertNotIn(endpoint, manager.connections.connected_peers)
+        self.assertNotIn(endpoint, set(manager.connections.iter_not_ready_endpoints()))
```

Comment:
I don't think we need these `set(...)` calls here.

Reply:
You're right, removing.

Reply:
Thanks!